lms/db/__init__.py

```diff
@@ -160,10 +146,6 @@ def includeme(config):
     # Create the SQLAlchemy engine and save a reference in the app registry.
     engine = make_engine(config.registry.settings)
     config.registry["sqlalchemy.engine"] = engine
-    init(engine)
```

Comment:
I think this should've been also inside the `if config.registry.settings["dev"]:`